packages/wdio-cli/src/utils.js

```diff
@@ -213,3 +212,30 @@ export async function missingConfigurationPrompt(command, message, useYarn = fal
 
     return await runConfig(useYarn, true)
 }
+
+export function getCapabilities(arg) {
+    const capabilities = getPlatformCaps(arg.option)
```

Comment:
let's merge these functions into one as I don't see a use for `getPlatformCaps` anywhere else

Reply:
Done, but for me now it looks a bit complex.